core/capabilities/ccip/configs/solana/chain_writer.go

```diff
@@ -360,130 +349,230 @@ func GetSolanaChainWriterConfig(offrampProgramAddress string, fromAddress string
 }
 
 func getOfframpAccountConfig(offrampProgramAddress string) chainwriter.Lookup {
-	return chainwriter.Lookup{PDALookups: &chainwriter.PDALookups{
-		Name: "OfframpAccountConfig",
-		PublicKey: chainwriter.Lookup{AccountConstant: &chainwriter.AccountConstant{
-			Address: offrampProgramAddress,
-		}},
-		Seeds: []chainwriter.Seed{
-			{Static: []byte("config")},
+	return chainwriter.Lookup{
+		PDALookups: &chainwriter.PDALookups{
+			Name: "OfframpAccountConfig",
+			PublicKey: chainwriter.Lookup{
+				AccountConstant: &chainwriter.AccountConstant{
+					Address: offrampProgramAddress,
+				},
+			},
+			Seeds: []chainwriter.Seed{
+				{Static: []byte("config")},
+			},
+			IsSigner:   false,
+			IsWritable: false,
 		},
-		IsSigner:   false,
-		IsWritable: false,
-	}}
+	}
 }
 
 func getAddressConstant(address string) chainwriter.Lookup {
-	return chainwriter.Lookup{AccountConstant: &chainwriter.AccountConstant{
-		Address:    address,
-		IsSigner:   false,
-		IsWritable: false,
-	}}
+	return chainwriter.Lookup{
+		AccountConstant: &chainwriter.AccountConstant{
+			Address:    address,
+			IsSigner:   false,
+			IsWritable: false,
+		},
+	}
 }
 
 func getFeeQuoterConfig(offrampProgramAddress string) chainwriter.Lookup {
```

Comment:
nit: the name of this function is a little confusing to me. I think something like `getFeeQuoterProgramAccount` would be better?